ansible/vars/icds/icds_public.yml

```diff
@@ -0,0 +1,195 @@
+SITE_HOST: 'icds-cas.gov.in'
+NO_WWW_SITE_HOST: 'icds-cas.gov.in'
+deploy_env: icds
+
+riakcs_s3_access_key: "{{ secrets.RIAKCS_S3_ACCESS_KEY }}"
+riakcs_s3_secret_key: "{{ secrets.RIAKCS_S3_SECRET_KEY }}"
+
+active_sites:
+  cchq_ssl: True
+  cchq_redirect: False
+  cchq_http: True
+  cchq_http_redirect: True
+  cchq_http_j2me: False
+  commtrack_ssl: False
+  commtrack_http: False
+  hq_status: False
+  tableau: False
+  icds_tableau: True
+  dhis: False
+  wiki: False
+  wiki_http: False
+  motech: False
+  motech1: False
+  motech2: False
+  riakcs: True
+  cas_ssl: False
```

Comment:
I think that this was because we need two proxies in softlayer. one for india and one for CAS. I think we can remove this once CAS has totally moved over. It should be `False` for now on icds and `True` for softlayer